Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -255,3 +270,20 @@
     - CrowbarBorg
     - FireExtinguisherBorg
     - HypoBorgStandard
+
+- type: entity
+  id: BorgERTModuleStandard
+  parent: [ BaseBorgModule, BaseProviderBorgModule ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)